components/header.tsx

```diff
@@ -1,18 +1,72 @@
 import { useLocale } from '@/lib/hooks'
+import { useState } from 'react'
 import OneZeroSkipToMainContent from './onezero-skip-to-main-content'
+import { Button } from './dataComponents/button'
+import { ChangeLangButton } from './dataComponents/changeLangButton'
+import { NextRouter, useRouter } from 'next/router'
 
-export default function Header() {
+export default function Header({ locale }: { locale: string }) {
```

Comment:
you don't need to pass locale from the parent you can just use `const { locale } =  useLocale()`

Reply:
fixed,
deleted {locale} as parent
use const { locale } = useLocale()